### PR TITLE
DRA-162 add footer

### PR DESCRIPTION
**Pull request by jesperlauridsen**
This should add the footer. Note, that the cooke settings button doesn't work, we have a seperate JIRA for that.

### CODE REVIEW

File: src/components/global/nav/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+	[column: string]: {
+		title: string;
+		links: {
+			[link: string]: {
+				title: string;
+				link: string;
+			};
+		};
+	};
+}
```
Comment (jorntx) [inline src/components/global/nav/Footer.vue:80]: Why type locally in the footer component  and not with the rest of the types?

File: src/components/global/nav/wc-header-menu.ts [hunk 2]
```diff
@@ -247,6 +247,7 @@ const MENU_COMPONMENT_STYLES = /*css*/ `
 	}
 
 	.rdl-logo {
+		/* TODO: We need a better way to get images into web components. */
```
Comment (jorntx) [inline src/components/global/nav/wc-header-menu.ts:250]: Agreed - can you make a ticket for it?

File: src/components/global/nav/Footer.vue [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+export default defineComponent({
+	name: 'Footer',
+	setup() {
+		const { t, messages, locale } = useI18n();
+		const currentLocaleMessages = ref(undefined as unknown as Footer);
+
+		onMounted(() => {
+			currentLocaleMessages.value = toRaw(messages.value[locale.value].footer) as Footer;
```
Comment (jorntx) [inline src/components/global/nav/Footer.vue:89]: So this is how it ended. Eyebrow raised but eyes not bleeding:)

**Review by jorntx — COMMENTED** [review]
Implementation looks good even with the "creative translation solution". 

I only have a few comments and a missing translation - see screenie :)

Screenshot 2024-03-14 at 07 55 16

File: src/components/global/nav/wc-header-menu.ts (hunk 2 above)
Comment (jesperlauridsen) [reply src/components/global/nav/wc-header-menu.ts:250]: Will do!

**Review by jorntx — APPROVED** [review]
Thanks for the changes - this is good to go now.

**Pull request merged**